src/Command/CommandBase.php

```diff
@@ -1089,7 +1089,13 @@ public static function getAliasCache() {
    */
   protected function doGetApplicationFromAlias($application_alias) {
     $acquia_cloud_client = $this->cloudApiClientService->getClient();
-    $acquia_cloud_client->addQuery('filter', 'hosting=prod:' . $application_alias);
+    $possible_hosting_prefixes = ['prod', 'devcloud', 'network', 'enterprise-g1', 'gardens'];
```

Comment:
Looks like you've already figured this out but FYI you can find the info for any sub with:

```
acli api:applications:find <UUID> | jq -r '.hosting.id' | tr -d \" | sed 's/:.*//'
```